index.html

```diff
@@ -0,0 +1,555 @@
+                             </div>
+                        </div>
+                        <div class="flex-item">
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/VW-V2g-vehicle-to-grid-power-electricity-ev-future-480x270.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                               <p>Researchers use AI to translate text found on ancient clay tablet</p>
+                               <small>Thomas Macaulay <i class="fa fa-circle"></i>14 hours ago</small> 
+                            </div>
+                        </div>
+                        <div class="flex-item">
+                            <img src="https://img-cdn.tnwcdn.com/image?fit=1280%2C720&url=https%3A%2F%2Fcdn0.tnwcdn.com%2Fwp-content%2Fblogs.dir%2F1%2Ffiles%2F2020%2F01%2Fhuawei.jpg&signature=98ef1c41c70220c7072fc3f4d8d92238" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>US looks to block Huawei from banks after Trump ban fails</p>
+                                <small>David Canellis <i class="fa fa-circle"></i>15 hours ago</small>
+                            </div>
+                        </div>
+                        <div class="flex-item">
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/LEGO-Super-Mario-796x417.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>LEGO Super Mario combines 2 staples of childhood</p>
+                                <small>Rachael Kaser <i class="fa fa-circle"></i>15 hours ago</small>
+                            </div>
+                        </div>
+                        <div class="flex-item">
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2015/12/Sundar-Pichai-Google-for-India-1200x600.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>No clue how to handle Corona Virus as a start-up founder? Here's what we are doing</p>
+                                <small>Patrick De Laive <i class="fa fa-circle"></i>15 hours ago</small>
+                            </div>
+                        </div>
+                        <div class="flex-item">
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/08/Discord-Go-Live-516x270.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>White House asks Silicon Valley for AI solutions to Corona Virus</p>
+                                <small>Thomas Macaulay <i class="fa fa-circle"></i>14 hours ago</small>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+
+                <div class="funding">
```

Comment:
- [ ] In the mobile version, you should only show three articles in this section. Yours has four. Please adjust this. 

- [ ] The section's articles should have 100% width when mobile. Please make this change.